opentelemetry/instrumentation/app/work.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "work"
+
+var (
+	meter         = otel.Meter(name)
+	tracer        = otel.Tracer(name)
+	workHistogram metric.Int64Histogram
+)
+
+func init() {
+	var err error
+	workHistogram, err = meter.Int64Histogram("example.histogram",
+		metric.WithDescription("Sample histogram"),
+		metric.WithUnit("1"))
```

Comment:
It would be nice for these to feel a tiny bit more real.  E.g. make up something that this is measuring, with an appropriate unit

Reply:
Or, if this is moved to the /multi endpoint, make this a histogram of the number of subrequests.

Reply:
If it is kept at the single endpoint, make it a histogram of the time slept.

Reply:
I can re-purpose the histogram to "measure" something. The rationale behind this was that we already have metrics like `server_duration_millis` that cover this use-case.

So this kind of work is actually a different use-case. Those random numbers being generated could represent different user-interactions in a production-app and the histogram shows the distribution for each of those interactions.

Reply:
Both your suggestions seem valid to me though.

I can do both and depending on how the doc write-up goes we can choose whichever one is more appropriate.

Reply:
FYI: For /multi endpoint the histogram tracking the number of requests will not always have a good distribution since the range of number of requests is quite low - 3 to 7. Even after explicit buckets, with sensible boundaries, there weren't many exemplars. But I feel its WAI.

Reply:
You can set the bucket boundaries if you want, rather than using the default.  E.g. buckets of [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].

Reply:
I thought these bucket sizes might become too fine grained, the current ones I have are [2,4,6,8,10] *(which are still a little fine-grained)*. 

Are they setting a good example for users?

Reply:
I think it is fine for now.  We can remove it once GMP supports exponential histograms